app/src/components/Common/Button/Button.tsx

```diff
@@ -20,12 +21,14 @@ export const Button = ({
   children,
   disabledResaon,
   className,
+  onLightBackground,
 }: IProps) => {
   const buttonStyle = classNames(style.tooltipHover, className, {
     [style.button]: !displayAsLink,
     [style.secondaryButton]: color === 'Secondary' && !displayAsLink,
     [style.primaryButton]: color === 'Primary' && !displayAsLink,
     [style.link]: displayAsLink,
+    [style.darkLink]: displayAsLink && onLightBackground,
```

Comment:
Kva er logikken i at den må vere `displayedAsLink` for å vere dark?

Reply:
Den må ikke være display as link for å være mørk, men det er 4 alternativer. Primær og sekunder-knapp (lys og mørk), og lys og mørk link-knapp